manifests/base/runner-manager/deployment.yaml

```
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: runner-manager
+  labels: {}
+spec:
+  selector: {}
+  template:
+    metadata: {}
```

Somebody complaining that those keys don't exist?
```suggestion
spec:
  template:
```

---

we are, but no need to specify an empty key IMHO, they will be set by the commonlabels anyway.